networkx/algorithms/tree/mst.py

```diff
@@ -15,9 +18,27 @@
     "maximum_spanning_edges",
     "minimum_spanning_tree",
     "maximum_spanning_tree",
+    "partition_spanning_tree",
+    "EdgePartition",
+    "SpanningTreeIterator",
 ]
 
 
+class EdgePartition(Enum):
```

Comment:
This looks identical to the definition in `branchings.py` - it would be best to define this class once and re-use it.

Reply:
The two definitions are identical. I've changed the `ArborescenceIterator` to use the one in `mst.py`.